client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -407,6 +408,24 @@ const Editor = (props: EditorProps): JSX.Element => {
       return accessibility;
     };
 
+    const isTabTrapped = () => !!(store.get('monacoTabTrapped') ?? true);
```

Comment:
```suggestion
    const isTabTrapped = () => !!store.get('monacoTabTrapped');
```
Same logic since `!!undefined === false` and if we've stored `"false"` in local storage, `store.get` will return `false`.

Reply:
I think this change changes the initial default behavior of the Tab key in the editor. The default is that the Tab key is trapped in the editor. So if the user has never used the hotkeys to change this behavior then `isTabTrapped` should return `true`. Thus, if the `monacoTabTrapped` key has never been set in local storage then `store.get()` will return `undefined` and this should cause `isTabTrapped` to return `true`. This change appears to cause it to return `false` in this situation.

Reply:
Yeah, you're right. I'm not sure what I thought was happening here.  Sorry about that!

What we could do is flip the variable we care about from `trapped` to `free`, then the logic works without having to handle the special case where the store returns `undefined`. I'll triple check it, but that seems to work nicely.

Reply:
```suggestion
    const isTabFree = () => !!store.get('monacoTabFree');
```

Reply:
Keeping the variable as `trapped`.